not-to-release/sources/newsgroup/groups.google.com_hiddennook_88969236563fa748_ENG_20050215_173600.xml.conllu

```diff
@@ -243,7 +243,7 @@
 18	pleased	pleased	ADJ	JJ	Degree=Pos	17	xcomp	17:xcomp	_
 19	that	that	SCONJ	IN	_	24	mark	24:mark	_
 20	the	the	DET	DT	Definite=Def|PronType=Art	22	det	22:det	_
-21	White	White	PROPN	NNP	Number=Sing	22	compound	22:compound	_
+21	White	White	ADJ	NNP	Degree=Pos	22	amod	22:amod	_
```

Comment:
Not necessarily saying we should change this, but oddly "White House" is sometimes given as a paradigm example of an English A+N compound, based on the Compound Stress Rule (cf. "Some white 'house")

Reply:
Yeah we definitely are not defining `compound` to include adjective-noun compounds.